node/status_node_services.go

```diff
@@ -347,7 +351,29 @@ func (b *StatusNode) wakuV2Service(nodeConfig *params.NodeConfig, telemetryServe
 			cfg.MaxMessageSize = nodeConfig.WakuV2Config.MaxMessageSize
 		}
 
-		w, err := wakuv2.New(nodeConfig.NodeKey, nodeConfig.ClusterConfig.Fleet, cfg, logutils.ZapLogger(), b.appDB, b.timeSource(), signal.SendHistoricMessagesRequestFailed, signal.SendPeerStats)
+		var nodeKey *ecdsa.PrivateKey
+		var err error
+		if nodeConfig.NodeKey != "" {
+			nodeKey, err = crypto.HexToECDSA(nodeConfig.NodeKey)
+			if err != nil {
+				return nil, fmt.Errorf("could not convert nodekey into a valid private key: %v", err)
+			}
+		} else {
+			nodeKeyStr := os.Getenv("WAKUV2_NODE_KEY")
```

Comment:
Maybe we could use a more concise env var. For example, in `nwaku-compose` we use `NODEKEY`:
https://github.com/waku-org/nwaku-compose/blob/bba6f8b9a5ad7c7758827a5b6109d4916f0c4cc0/.env.example#L15